docs/runbooks/stripe-hardening-rollout.md

```
@@ -0,0 +1,192 @@
+
+## Phase 1 — Hardening deploy (no behavior change)
+
+**Preconditions:** PR 1 merged. No env changes needed — the code falls back to the
+existing single `STRIPE_WEBHOOK_SECRET`.
+
+**Steps (demo, then prod):**
+
+1. Deploy backend as usual (`./deploy.sh update` on the target host); migrations run on
+   deploy (new `StripeWebhookEvent` table + the pruning beat task).
+2. Verify the migration applied:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)